modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/core/useIE11Compatibility.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {useLayoutEffect} from 'react';
+
+export default function useIE11Compatibility() {
+	useLayoutEffect(() => {
+		if (Liferay.Browser?.isIe()) {
+			const oldRemoveChild = window.HTMLElement.prototype.removeChild;
+
+			window.HTMLElement.prototype.removeChild = function () {
+				try {
+					return oldRemoveChild.apply(this, arguments);
+				}
+				catch (error) {
+					if (
+						process.env.NODE_ENV === 'development' &&
+						!!error &&
+						!!error.message &&
+						error.message === 'NotFoundError'
+					) {
+						console.warn('IE NotFoundError handled');
+					}
+					else {
+						throw error;
```

Comment:
Why re-`throw` here? If I'm not in `development` mode (ie. all customers and all production sites), it seems like we'll still be `throw`-ing.

Reply:
Totally right, I'll remove that, what I wanted is to avoid logging if we are not in development

Reply:
Re pushed with a fix :)